### PR TITLE
added navbar footer card and category button

### PR DESCRIPTION
**Pull request by jadambre**

### CODE REVIEW

File: frontend/src/App.jsx
Comment (mlAgostini) [inline frontend/src/App.jsx:1]: missing div closure

File: frontend/src/components/Card/CardStyle.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export default styled.div`
+  display: flex;
+  justify-content: space-around;
+
+  img {
```
Comment (fantinerudent) [inline frontend/src/components/Card/CardStyle.jsx:7]: as i explained, styled div should not hold another atribute

**Review by mlAgostini — CHANGES_REQUESTED** [review]

File: frontend/src/components/CategoryButton/CategoryButtonStyle.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+export default styled.div`
+  display: flex;
+  justify-content: flex-start;
+
+  img {
```
Comment (fantinerudent) [inline frontend/src/components/CategoryButton/CategoryButtonStyle.jsx:7]: same

File: frontend/src/components/Footer/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import SFooter from "./FooterStyle";
+import Footerimg from "../../assets/Footerimg.svg";
```
Comment (fantinerudent) [inline frontend/src/components/Footer/Footer.jsx:2]: if it is a variable name, do not use an upercase , it is not a react component

File: frontend/src/components/Footer/FooterStyle.jsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+
+export default styled.footer`
+  img {
```
Comment (fantinerudent) [inline frontend/src/components/Footer/FooterStyle.jsx:4]: same

File: frontend/src/components/Navbar/NavbarStyle.jsx [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+
+export default styled.nav`
+  img {
```
Comment (fantinerudent) [inline frontend/src/components/Navbar/NavbarStyle.jsx:4]: same

File: frontend/src/pages/Home.jsx [hunk 6]
```diff
@@ -1,35 +1,17 @@
...
-//         </a>
-//       </p>
-//     </header>
-//   );
-// }
+export default function Home() {
+  return (
+    <SHome>
+      <Navbar />
+      <h1 />
```
Comment (fantinerudent) [inline frontend/src/pages/Home.jsx:11]: useless H1

File: frontend/src/pages/HomeStyle.jsx [hunk 7]
```diff
@@ -0,0 +1,3 @@
+import styled from "styled-components";
+
```
Comment (fantinerudent) [inline frontend/src/pages/HomeStyle.jsx:2]: useless file

**Review by fantinerudent — CHANGES_REQUESTED** [review]

**Pull request closed without merging**